robottelo/entities.py

```diff
@@ -825,14 +827,10 @@ class GPGKey(
         orm.Entity, orm.EntityReadMixin, orm.EntityDeleteMixin,
         orm.EntityCreateMixin):
     """A representation of a GPG Key entity."""
-    organization = entity_fields.OneToOneField('Organization', required=True)
-    location = entity_fields.OneToOneField('Location', null=True)
```

Comment:
`location` is not mentioned in the API docs on either upstream or downstream systems.